src/Api/Program.cs

```diff
@@ -29,35 +29,40 @@
 
 builder.WebHost.ConfigureKestrel(c => c.AddServerHeader = false);
 builder.Host.UseSerilog((ctx, sp, config) =>
-{
-    config
-        .ReadFrom.Configuration(ctx.Configuration)
-        .ReadFrom.Services(sp)
-        .Enrich.FromLogContext()
-        .Enrich.WithMachineName()
-        .Enrich.WithEnvironmentName()
-        .WriteTo.Console();
-
-    if (builder.Environment.IsDevelopment())
-    {
-        config.WriteTo.Seq("http://localhost:5341");
-    }
-
-    var ddApiKey = Environment.GetEnvironmentVariable("DD_API_KEY");
-    if (!string.IsNullOrEmpty(ddApiKey))
     {
-        var ddSite = Environment.GetEnvironmentVariable("DD_SITE") ?? "datadoghq.eu";
-        var ddUrl = $"https://http-intake.logs.{ddSite}";
-        var ddConfig = new DatadogConfiguration(ddUrl);
+        config
+            .ReadFrom.Configuration(ctx.Configuration)
+            .ReadFrom.Services(sp)
+            .Enrich.FromLogContext()
+            .Enrich.WithMachineName()
+            .Enrich.WithEnvironmentName()
+            .WriteTo.Console();
+
+        if (builder.Environment.IsDevelopment())
+        {
+            config.WriteTo.Seq("http://localhost:5341");
+        }
 
+        var ddApiKey = Environment.GetEnvironmentVariable("DD_API_KEY");
         if (!string.IsNullOrEmpty(ddApiKey))
         {
-            config.WriteTo.DatadogLogs(
-                ddApiKey,
-                configuration: ddConfig);
+            var ddSite = Environment.GetEnvironmentVariable("DD_SITE") ?? "datadoghq.eu";
+            var ddUrl = $"https://http-intake.logs.{ddSite}";
+            var ddConfig = new DatadogConfiguration(ddUrl);
+
+            if (!string.IsNullOrEmpty(ddApiKey))
+            {
+                config.WriteTo.DatadogLogs(
+                    ddApiKey,
+                    configuration: ddConfig);
+            }
         }
-    }
-});
+    },
+    false,
+    // Pass log events down to other logging providers (e.g. Microsoft) after Serilog, so
+    // that they can be processed in a uniform way in tests.
+    true
```

Comment:
This was required because Serilog overshadows other logging providers and there's no easy way (that I found) to append another sink to an existing Serilog configuration for use in tests.